apiary.apib

```diff
@@ -5396,3 +5396,26 @@ From now on you won't be able to sign in to the **Direct Access** for specific p
             X-StorageApi-Token: your_token
 
 + Response 204
+
+###Reset credentials for backend [PUT /v2/storage/direct-access/{backend}/reset-password]
+Resets password for **Direct Access** backend. In response you will get url address for create new password.
+The password will be reset only for the project for which the **storage api token** is created.
+
++ Parameters
+    + include (required, enum[string]) - Backend for which you want to reset the password.
```

Comment:
Tohle určitě není správně, ne? Ten parametr se nejmenuje include, ale backend! Enum bohužel neumí říct, že tam může být jen jedna hodnota (https://apiblueprint.org/documentation/mson/specification.html#43-nested-member-types) ale i tak bych ho tam asi nechal. 

Default tam ovšem není. Když ho do URL nedáš, tak to nefunguje.